.prettierignore

```diff
@@ -1,2 +1,4 @@
 # The GH actions don't seem to compile and verify themselves well when Prettier is applied to them
 .github/actions/javascript/**/index.js
+desktop/dist/**/*.js
+dist/**/*.js
```

Comment:
can we combine these conditions into one like this

`**/dist/**/*.js`

we want to ignore all `js` files under all `dist` directories, right?

Reply:
Oh, sneaky. I like that, but it _could_ have unintended side-effects that would be hard to catch (like if someone just happened to create a `src/libs/dist` folder).

Reply:
that makes sense good to be explicit about these things 👍